core/modules/widgets/widget.js

```diff
@@ -229,7 +229,7 @@ Widget.prototype.getStateQualifier = function(name) {
 			}
 			node = node.parentWidget;
 		}
-		var value = $tw.utils.hashString(output.join(""));
+		var value = Math.abs($tw.utils.hashString(output.join(" ")));
```

Comment:
IMO you can't do that. You are deliberately combining 2 different values to 1. There is a reason why 2 different strings should result in 2 different hashes.

Reply:
replace the `-` with `x` or `0` and you are probably good to go

Reply:
thanks @pmario - I've changed it